Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2292,6 +2297,8 @@ extension BrowserViewController: WKNavigationDelegate {
                     UIApplication.shared.openURL(url)
                 }))
                 present(alert, animated: true, completion: nil)
+
+                LeanplumIntegration.sharedInstance.track(eventName: .openedTelephoneLink)
```

Comment:
Removing it and will have @bkmunar fix: https://bugzilla.mozilla.org/show_bug.cgi?id=1367969